wordpress-mobile/gutenberg-mobile
# Update: Image Block Size picker

This updates the Image block size picker. 

To test:
See the Gutenberg PR, https://github.com/WordPress/gutenberg/pull/31963.

PR submission checklist:
